testing-styleguide/README.md

```diff
@@ -0,0 +1,495 @@
+<!--
+ Copyright (c) 2022 Joao Mello (joao.mello@geekhunter.com.br)
+ 
+ This software is released under the MIT License.
+ https://opensource.org/licenses/MIT
+-->
+
+<p align="center">
+  <img alt="logo" src="/docs/logo.png" width="200">
+</p>
+
+# ♾️ The Geek Testing Style Guide
+
+Welcome to our testing style guide!
+
+The goal of this guide is to help our team to think about tests, write them, and review someone else's code taking them into account.
+
+`#letscode 😎👩‍💻👨‍💻`
+
+# :pushpin: Summary
+
+* [Introduction](#introduction)
+* [Concepts](#concepts)
+* [Strategies](#strategies)
+* [In practice: Rails](#in-practice-rails)
+* [In practice: React](#in-practice-react)
```

Comment:
missing this section?

Reply:
will be added in a following PR by @VictorSpulveda